symphony-bdk-extensions/symphony-group-extension/src/main/java/com/symphony/bdk/ext/group/auth/OAuth.java

```diff
@@ -0,0 +1,25 @@
+package com.symphony.bdk.ext.group.auth;
+
+import com.symphony.bdk.http.api.auth.Authentication;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.apiguardian.api.API;
+
+import java.util.Map;
+import java.util.function.Supplier;
+
+@Getter
+@Setter
+@API(status = API.Status.INTERNAL)
+public class OAuth implements Authentication {
+
+  private Supplier<String> bearerTokenSupplier = () -> "";
+
+  @Override
+  public void apply(final Map<String, String> headerParams) {
+    if (this.bearerTokenSupplier != null) {
```

Comment:
Is there a use case for calling `setBearerTokenSupplier(null)`? If not and since there is already a default value, I would even remove the `if` statement.

Reply:
Good catch 👍 I even managed to make it `final`